packages/local_auth/local_auth_windows/example/README.md

```diff
@@ -0,0 +1,8 @@
+# local_auth_example
+
+Demonstrates how to use the local_auth plugin.
+
+## Getting Started
```

Comment:
Sorry, missed this before. This whole section is boilerplate and should be removed. (I need to scrub the whole repo for it and clean it all up.)

Reply:
Fixed.